testing-with-jest.test.js

```diff
@@ -6,22 +6,84 @@
 // $ yarn jest
 
 
-// --------------------1a) Create a test for a function called addThemUp that takes two numbers as arguments and returns the sum.
-
-
-
-
-
-// --------------------1b) See the test fail. THEN write the code to make the test pass.
-
-
-
-
-
-// --------------------2a) Create a test for a function called triangleArea that takes the base and the height of a triange and returns the area. The area of a triange is the base times the height divided by two.
+// -----COMPLETE------1a) Create a test for a function called addThemUp that takes two numbers as arguments and returns the sum.
+// almost didnt add yarn to jest before running haha
+// write out basic form of the tests
+// // create sentence saying what I want "two numbers total"
+// // needs and argument and a return
+describe("addThemUp",() =>{
+  test("takes two numbers and returns the sum",()=>{
+    expect(addThemUp(2 + 3)).toEqual(5)
+  })
+})
+// needs a function to operate, DO NOT NEED console.log();
+// just yarn jest in terminal
+const addThemUp = () =>{
+  return "sum"
+}
+//
+// // --------------------1b) See the test fail. THEN write the code to make the test pass.
+
+describe("addThemUp",() =>{
+  test("takes two numbers and returns the sum",()=>{
+    expect(addThemUp(2 + 3)).toEqual("sum")
+  })
+})
+
+const addThemUp = () =>{
+  return "sum"
+// }
```

Comment:
This, doesn't actually do anything. It also seems to be a repeat of the test above it. 

The format for these challenges was to write the test on 1a, then on 1b, you write the function logic.

Reply:
Ya I seemed to have not understood the questions at all...I swear I reread them multiple times over the weekend, just was mixing up the terms and names obviously, I felt great that it passed, but its not a working code, so back to the drawing board.